src/engine/qcommon/files.cpp

```diff
@@ -639,7 +639,19 @@ void FS_LoadAllMapMetadata()
 {
 	for (const auto& pak: FS::GetAvailableMapPaks()) {
 		try {
-			FS::PakPath::LoadPakPrefix(pak, va("meta/%s/", pak.name.substr(4).c_str()));
+			std::string pak_map_prefix("map-");
+			// If pak name is long enough to have “map-” prefix
+			// and pak name without "map-" prefix is not empty string.
+			if (pak.name.length() > pak_map_prefix.length())
+			{
+				// If pak name starts with "map-" prefix
+				if (pak.name.rfind(pak_map_prefix, 0) == 0)
```

Comment:
Out of curiosity, why not just `find(pak_map_prefix)` ?

https://en.cppreference.com/w/cpp/string/basic_string/find

Reply:
Right, `rfind` would do “startswith without other occurrences”, while `find` allows package to be named `map-map-map_0.dpk`, which is legit. I'm fixing it.